static-responses/README.md

```diff
@@ -0,0 +1,64 @@
+# Static Responses
+
+There are situations where no backend requests are sent for an endpoint.
+Nevertheless, a response is needed. So a `response` block can exist in an
+`endpoint` block without any `proxy` or `request` blocks.
+
+Consider the following example, in which the client is instructed to perform a
+redirect via a `303` status code an a `location` header specifying the redirect
+target:
+
+```hcl
+    endpoint "/redirect" {
+      response {
+        status = 303
+        headers = {
+          location = "https://www.example.com/"
+        }
+      }
+    }
+```
+
+Call Couper:
+
+```shell
+$ curl -i http://localhost:8080/redirect
+HTTP/1.1 303 See Other
+Location: https://www.example.com/
+...
+
+```
+
+Another example is a simple userinfo endpoint, similar to the one defined by
+[OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo).
+The endpoint is protected by the "JWTToken" access control. The token claims
+are reflected to the client.
+
+```hcl
+    endpoint "/userinfo" {
+      access_control = ["JWTToken"]
+      response {
+        status = 200
+        json_body = req.ctx.JWTToken
+      }
+    }
+...
+definitions {
+  jwt "JWTToken" {
+    header = "Authorization"
+    signature_algorithm = "RS256"
+    key_file = "../jwt-access-control/pub.pem"
```

Comment:
that doesn't work with the `docker run` documented in the top level readme.

all examples should be self contained.